packages/components/src/forms/Input/Input.android.tsx

```
@@ -1,13 +1,76 @@
 import TextInput from './TextInput';
 
+import type { SizeVariantSpreadFunction } from '@tamagui/core';
 import type { InputExtraProps } from 'tamagui';
 
+export const textAreaSizeVariant: SizeVariantSpreadFunction<any> = (
+  val = '$true',
+  extras,
+) => {
+  const { props } = extras;
+  const buttonStyles = getButtonSized(val, extras);
+  const fontStyle = getFontSized(val as any, extras)!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)